db/migrate/20221027150350_add_smod_to_capital_items.rb

```diff
@@ -0,0 +1,8 @@
+class AddSmodToCapitalItems < ActiveRecord::Migration[7.0]
+  def change
+    add_column :capital_items, :subject_matter_of_dispute, :boolean, default: false
+    add_column :vehicles, :subject_matter_of_dispute, :boolean, default: false
+    add_column :properties, :subject_matter_of_dispute, :boolean, default: false
+    add_column :capital_summaries, :subject_matter_of_dispute_disregard, :decimal, default: "0.0", null: false
```

Comment:
I think nullable: true is a good additional property to set here. Do we need default: false? (I can believe that the answer is yes, I'm just checking)

Reply:
Fair point. The database is a mish-mash of user-provided data and calculated data. From the calculation perspective an asset is either treated as smod or it isn't, but a user can either tell us an asset is smod, or tell us it isn't, or not tell us at all. It's probably more valuable to store the user-provided information in the database and infer the calculation information from it.